packages/wrangler/src/index.ts

```diff
@@ -49,6 +48,7 @@ import { initHandler, initOptions } from "./init";
 import "./kv";
 import "./workflows";
 import "./user/commands";
+import "./docs";
```

Comment:
I think the import order matters in terms of which commands are displayed first on the help message and I though the docs command was the first one..? Can you help confirming?

Reply:
sort of - right now register.registerNamespace determines the order, because it needs to be interleaved with the non-defineCommand commands (which is why the help tests are passing). i think eventually it will pull from the import order and then we can get rid of registerNamespace. I forgot about that tho, will reorder.